packages/muelu/src/Misc/MueLu_InterfaceMappingTransferFactory_def.hpp

```diff
@@ -0,0 +1,38 @@
+#ifndef MUELU_INTERFACEMAPPINGTRANSFERFACTORY_DEF_HPP_
+#define MUELU_INTERFACEMAPPINGTRANSFERFACTORY_DEF_HPP_
+
+#include "MueLu_InterfaceMappingTransferFactory_decl.hpp"
+
+namespace MueLu
+{
+
+template <class LocalOrdinal, class GlobalOrdinal, class Node>
+InterfaceMappingTransferFactory<LocalOrdinal, GlobalOrdinal, Node>::InterfaceMappingTransferFactory()
```

Comment:
Can be removed if the qualifier `default` is used in the declaration of the constructor.